lib/Backends/DeviceManagers.cpp

```diff
@@ -61,7 +62,7 @@ DeviceManager *DeviceManager::createDeviceManager(BackendKind backendKind,
   case BackendKind::CPU:
     return createCPUDeviceManager(name);
   default:
-    GLOW_UNREACHABLE("not supported backend");
+    return new InlineDeviceManager(backendKind, name);
```

Comment:
Could you add a comment here saying that we create a dummy device manager here as a fallback solution, but people should better implement their own device manager with proper semantics.